docker/Dockerfile-osc-base

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+
+MAINTAINER Arvind Nadendla <arvindn05@gmail.com>
+
+ENV http_proxy http://proxy-chain.intel.com:911
```

Comment:
proxy cannot be hardcoded, rather it should be placed in configuration. or we add a note, so that user can change to no-proxy or relevant proxy depending on the need.

Reply:
Done, I created a new proxy.sh file, where user can configure proxy settings